<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartitionWidget</class>
 <widget class="QWidget" name="PartitionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Legend</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="description">
        <property name="text">
         <string>Change field and colors for stacked histograms. Double click to edit color. Single click on selected to edit label.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="minLayout">
        <item>
         <widget class="QCheckBox" name="minCheckBox">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="minSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="maxLayout">
        <item>
         <widget class="QCheckBox" name="maxCheckBox">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="maxSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="binLayout">
        <item>
         <widget class="QCheckBox" name="binCheckBox">
          <property name="text">
           <string>Bins:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="binSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>binCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>binSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
